games-strategy/wesnoth/wesnoth-1.18.2.recipe

```diff
@@ -0,0 +1,175 @@
+SUMMARY="Turn-based tactical strategy game with a high fantasy theme"
+DESCRIPTION="Build up a great army, gradually turning raw recruits into \
+hardened veterans. In later games, recall your toughest warriors and form a \
+deadly host whom none can stand against! Choose units from a large pool of \
+specialists, and hand-pick a force with the right strengths to fight well on \
+different terrains against all manner of opposition.
+
+Wesnoth has many different sagas waiting to be played. Fight to regain the \
+throne of Wesnoth, of which you are the legitimate heir... step into the \
+boots of a young officer sent to guard a not-so-sleepy frontier outpost... \
+lead a brutal quest to unite the orcish tribes... vanquish a horde of undead \
+warriors unleashed by a foul necromancer, who also happens to have taken your \
+brother hostage... guide a band of elvish survivors in an epic quest to find \
+a new home... get revenge on the orcs by using the foul art of necromancy...
+
+There are at least two hundred unit types, sixteen races, six major factions, \
+and hundreds of years of history. The world of Wesnoth is absolutely huge and \
+only limited by your creativity — make your own custom units, create your own \
+maps, and write your own scenarios or even full-blown campaigns. You can also \
+challenge up to eight friends—or strangers—and fight in epic multiplayer \
+fantasy battles."
+HOMEPAGE="http://wesnoth.org/"
+COPYRIGHT="2003-2024 The Battle for Wesnoth Project"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://sourceforge.net/projects/wesnoth/files/wesnoth-1.18/wesnoth-$portVersion/wesnoth-$portVersion.tar.bz2"
+CHECKSUM_SHA256="5f9f6d6105d48cf614eb06cf72273d62931dcb5d4982d75e851c3fa2fa70553a"
+PATCHES="wesnoth-$portVersion.patchset"
+ADDITIONAL_FILES="
+	wesnoth.rdef.in
+	wesnoth-postinstall.sh
+	"
+POST_INSTALL_SCRIPTS="$relativePostInstallDir/wesnoth-postinstall.sh"
+
+ARCHITECTURES="?all !x86_gcc2 x86 x86_64"
```

Comment:
```suggestion
ARCHITECTURES="all !x86_gcc2"
```

Reply:
Haven't tested it on arm or ppc. The way I wrote it is more future-proof :)

Reply:
It's the default way atm, can't force you to do it though.

Reply:
Sorry, honestly I don't think it's a clever default, it kind of assumes an arm or ppc version will never see the light of day

Reply:
How is that, there are no buildmasters available for those arch's, so builders should check them on their own and comment/disable them when something is broken.
Also, please don't resolve all things every time, it may be resolved in your eyes, but it's harder to track which ones "I" marked as changes needed or not. :)

Reply:
Nobody's going to check all packages for all architectures. Now there's no buildmaster for arm or ppc, but what about in a bunch of years? The syntax I'm using changes nothing now, reflects what I know about the build process (as in, I have no clue if it compiles on those) and lets people know those architectures haven't been tested, currently one could read the line and think "ah, so ppc and arm have been tested on this", which is untrue.

Reply:
Anyway, since it's the standard and the recipe is going to be obsolete in a year or two I've amended architectures as suggested